lib/sync/endpoint.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Adds an http based signaling server for real time collaboration.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Reads the contents of $fd and returns them unserialized.
+ *
+ * @access private
+ * @internal
+ *
+ * @param integer $fd      A file descriptor.
```

Comment:
`$fd` is not an integer.
```suggestion
 * @param resource $fd  A file descriptor.
```